movies/src/components/filter/SortBy.js

```diff
@@ -1,22 +1,22 @@
 import React from 'react';
 import Dropdown from '../common/dropdown/Dropdown';
+import Constants from '../constants';
 
-const filters = [
-  {
-    title: 'release date',
-    id: 'f1',
-  },
-  {
-    title: 'title',
-    id: 'f2',
-  },
-];
+export default function SortBy(props) {
+  const { updateMovies } = props;
+  const sortBy = (sortType) => {
```

Comment:
method can be moved outside `SortBy` component

Reply:
Fixed